### PR TITLE
Add info and fix command description [ci-skip]

### PR DESCRIPTION
**Pull request by siaw23**

### CODE REVIEW

File: guides/source/command_line.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ $ rails new commandsapp
         run  bundle install
 ```
 
-Rails will set you up with what seems like a huge amount of stuff for such a tiny command! You've got the entire Rails directory structure now with all the code you need to run our simple application right out of the box.
+Rails will set us up with what seems like a huge amount of stuff for such a tiny command! We've got the entire Rails directory structure now with all the code we need to run our simple application right out of the box.
```
Comment (jonathanhefner) [inline guides/source/command_line.md:45]:
```suggestion
Rails will set up what seems like a huge amount of stuff for such a tiny command! We've got the entire Rails directory structure now with all the code we need to run our simple application right out of the box.
```

File: guides/source/command_line.md [hunk 2]
```diff
@@ -42,13 +42,13 @@ $ rails new commandsapp
...
 
-If you wish to skip some files or components from being generated, you can append the following arguments to your `rails new` command:
+If you wish to skip some files from being generated or skip some libraries, you can append any of the following arguments to your `rails new` command:
 
 | Argument                | Description                                                 |
 | ----------------------- | ----------------------------------------------------------- |
-| `--skip-git`            | Skip .gitignore file                                        |
+| `--skip-git`            | Skip git init, .gitignore, and .gitattributes                |
```
Comment (jonathanhefner) [inline guides/source/command_line.md:51]:
```suggestion
| `--skip-git`            | Skip git init, .gitignore, and .gitattributes               |
```

File: guides/source/command_line.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ If you wish to skip some files or components from being generated, you can appen
 | `--skip-system-test`    | Skip system test files                                      |
 | `--skip-bootsnap`       | Skip bootsnap gem                                           |
 
+INFO: These are just some of the options you can append to `rails new`. For a comprehensive list of options, type `rails new --help`.
```
Comment (jonathanhefner) [inline guides/source/command_line.md:68]: Let's skip the `INFO:` callout here.  I don't think it adds anything, and too many callouts in a guide can be distracting.

```suggestion
These are just some of the options that `rails new` accepts. For a full list of options, type `rails new --help`.
```

**Review by jonathanhefner — COMMENTED** [review]
Some small tweaks, but looks good! :+1: 

When you're happy with all the changes, would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?

**Comment by siaw23** [comment]
I think I'm happy with the changes now :) @jonathanhefner

**Comment by jonathanhefner** [comment]
Thank you, @siaw23! :tada:

**Pull request merged**